java-test-samples/apigw-lambda-list-s3-buckets-spock/README.md

```diff
@@ -0,0 +1,240 @@
+# Java Test Samples Starter Project
```

Comment:
I had the same mixed feelings on this - I figured if I was looking / searching for examples with Spock this would be the least path of resistance for me as I wouldn't typically expect a mix of tests that tested the same outcomes.

My second consideration was for Java developers that may not have familiarity with Spock and adding a groovy package alongside the JUnit examples may add confusion in terms of dependencies etc. Happy to merge to the existing codebase if it makes a more coherent solution.

Reply:
@mridehalgh Thank you for the explanation. I agree with the sentiment. However, I strongly believe maintaining in the existing code will be better because of following reasons:
- The sample apps are created based on a serverless use case instead of a focus on a specific test framework
- The application code is repeated. Therefore, not DRY.
- Maven allows `profiles` which I believe can be used to have selective dependency management when it comes to JUnit and Spock. Keep JUnit profile as `default` and activate a new profile to show case Spock test on-demand.
- Lastly, the best place to elaborate your thinking process will be in the README file of the existing app. Create new section on Spock and elaborate all of the above including how Spock tests work.

I believe you can achieve more by doing less. Let me know what you think.